extensions/amp-form/0.1/amp-form.js

```diff
@@ -209,7 +209,7 @@ export class AmpForm {
         this.actions_.trigger(this.form_, 'submit-error', null);
         this.setState_(FormState_.SUBMIT_ERROR);
         this.renderTemplate_(error.responseJson || {});
-        rethrowAsync('Form submission failed:', error);
+        rethrowAsync(user().createError('Form submission failed:', error));
```

Comment:
Hmm. Normally a point of `rethrowAsync` is to preserve the user/dev flag. This might be correct, but to confirm: why are we converting this error to a user error?